src/components/Footer/Footer.module.css

```diff
@@ -38,9 +39,20 @@
 
 .subtitle {
   position: absolute;
-  width: max-content;
   top: 4rem;
   left: 5rem;
+  width: max-content;
+  max-width: 50vw;
+  word-wrap: break-word;
+}
+@media screen and (min-width: 40rem) {
+  .subtitle {
+    max-width: 40vw;
+  }
+
+  .main {
+    margin-bottom: unset;
+  }
```

Comment:
suggestion(blocking): I would suggest trying to utilize `100%` here rather than linking this to viewport.
We should also be able to utilize more space on mobile that way, as the navigation gets broken down into a new row in that view

Reply:
example from preview: 
![image](https://github.com/user-attachments/assets/12819f40-688b-46f3-a643-45a9caac90d8)